### PR TITLE
Update curl example in "Command line crash course"

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates the examples for using `curl` across all locales, updating them to use localized links instead.

### CODE REVIEW

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ ls | wc -l
 
 但是，这个 URL 是页面的旧位置。如果您在一个新的浏览器标签中输入它，您将 (最终) 被重定向到[https://developer.mozilla.org/en-US/docs/Web/API/fetch](/zh-CN/docs/Web/API/fetch).
 
-因此，如果您使用 curl 请求 `https://developer.mozilla.org/docs/Web/API/fetch`，则不会得到输出。现在就试试：
+因此，如果您使用 curl 请求 `https://developer.mozilla.org/zh-CN/docs/Web/API/fetch`，则不会得到输出。现在就试试：
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:275]:
```suggestion
因此，如果您使用 curl 请求 `https://developer.mozilla.org/docs/Web/API/fetch`，则不会得到输出。现在就试试：
```

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 2]
```diff
@@ -272,7 +272,7 @@ ls | wc -l
...
 
 ```bash
 curl https://developer.mozilla.org/zh-CN/docs/Web/API/fetch
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:278]:
```suggestion
curl https://developer.mozilla.org/docs/Web/API/fetch
```

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 3]
```diff
@@ -285,27 +285,27 @@ curl https://developer.mozilla.org/zh-CN/docs/Web/API/fetch
 尝试运行以下代码，您将看到在到达最终页面之前，实际上发生了三次重定向
 
 ```bash
-curl https://developer.mozilla.org/docs/Web/API/fetch -L -I | grep location
+curl https://developer.mozilla.org/zh-CN/docs/Web/API/fetch -L -I | grep location
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:288]:
```suggestion
curl https://developer.mozilla.org/docs/Web/API/fetch -L -I | grep location
```

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 4]
```diff
@@ -285,27 +285,27 @@ curl https://developer.mozilla.org/zh-CN/docs/Web/API/fetch
...
 ```
 
 输出应该是这样的 (`curl` 首先会输出一些下载计数器之类的东西):
 
 ```bash
-location: /en-US/docs/Web/API/fetch
+location: /zh-CN/docs/Web/API/fetch
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:294]:
```suggestion
location: /en-US/docs/Web/API/fetch
```

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 5]
```diff
@@ -285,27 +285,27 @@ curl https://developer.mozilla.org/zh-CN/docs/Web/API/fetch
...
 ```
 
 尽管有些做作，我们可以把这个结果做得更深入一点，并变换 `location:` 行内容，将基本的起点添加到每个起点的开始，这样我们就可以打印出完整的 url。为此，我们将在混合中添加 awk(它是一种类似于 JavaScript、Ruby 或 Python 的编程语言，只是要老得多 !)
 
 尝试运行这个：
 
 ```bash
-curl https://developer.mozilla.org/docs/Web/API/fetch -L -I | grep location | awk '{ print "https://developer.mozilla.org" $2 }'
+curl https://developer.mozilla.org/zh-CN/docs/Web/API/fetch -L -I | grep location | awk '{ print "https://developer.mozilla.org" $2 }'
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:302]:
```suggestion
curl https://developer.mozilla.org/docs/Web/API/fetch -L -I | grep location | awk '{ print "https://developer.mozilla.org" $2 }'
```

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 6]
```diff
@@ -285,27 +285,27 @@ curl https://developer.mozilla.org/zh-CN/docs/Web/API/fetch
...
 ```
 
 最终的输出应该是这样的
 
 ```bash
-https://developer.mozilla.org/en-US/docs/Web/API/fetch
+https://developer.mozilla.org/zh-CN/docs/Web/API/fetch
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:308]:
```suggestion
https://developer.mozilla.org/en-US/docs/Web/API/fetch
```

**Review by cw118 — APPROVED** [review]
LGTM for l10n-fr, thank you @queengooborg !

**Review by yin1999 — COMMENTED** [review]
Hi @queengooborg. This is a redirection example. So we may "need" to use `https://developer.mozilla.org/docs/Web/API/fetch`. As this would make the server to response a "location" header.

**Review by cw118 — CHANGES_REQUESTED** [review]
After taking a closer look at the contents/subject of the page, I agree with @yin1999, though it seems that the French version will need an update.

**Review by cw118 — COMMENTED** [review]
Just one small change, sorry for not being clear before 😖

The French content will need to be updated vs. en-US later, but I read the page more attentively and this example (for now) intentionally does not specify a locale.

**Review by cw118 — APPROVED** [review]
Thanks once again 🙏

**Review by yin1999 — APPROVED** [review]
LGTM for zh. Thanks 😊

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja`. Thank you!

**Pull request merged**